pinot-plugins/pinot-input-format/pinot-avro/pom.xml

```diff
@@ -37,15 +37,6 @@
     <phase.prop>package</phase.prop>
   </properties>
 
-  <profiles>
```

Comment:
I don't see any new module for these pluggable modules. Will they come in the following PRs or is there any other plan?

Reply:
no plan, we just don't have to release those modules in jdk8 really.

Reply:
presto-pinot-driver has transitive test dependencies on those jars, so we actually don't need to release those jars with JDK 8.

Reply:
But we do have some code which is maintained by Pinot only. E.g. in `pinot-avro-base` module, we have `AvroRecordReader` which is only for Pinot. How do we deal with this code?

Reply:
right, so for any dependencies that presto is depending on, we need to make those have a jdk8 version.
For module like `pinot-avro-base`, it's not required the compile dependencies of presto-pinot-driver but only test dependency, so this time I don't include them.
<img width="608" alt="image" src="https://user-images.githubusercontent.com/1202120/156240819-179238bb-0f49-4569-9d13-c766d3a69d75.png">

Reply:
> But we do have some code which is maintained by Pinot only. E.g. in `pinot-avro-base` module, we have `AvroRecordReader` which is only for Pinot. How do we deal with this code?

We don't need to do anything with this code.